src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -310,7 +310,10 @@ def decompress(self, model_path: str, model: Module):
         model_path = get_safetensors_folder(model_path)
         sparse_decompressed = False
 
-        if self.sparsity_compressor is not None:
+        if (
+            self.sparsity_compressor is not None
+            and self.sparsity_config.format != "dense"
```

Comment:
Is there an enum you could lookup instead of raw value?

Reply:
```suggestion
            and self.sparsity_config.format != CompressionFormat.dense
```

Reply:
I think we might want to do 
`>>> CompressionFormat.dense == "dense"
False
>>> CompressionFormat.dense.value == "dense"
True`

Reply:
Accepted!